jquery.confirm.js

```diff
@@ -69,7 +69,10 @@
         // Default options
         var settings = $.extend({}, $.confirm.options, {
             confirm: function () {
-                if (dataOptions.submitForm){
+                if (dataOptions.submitForm
+                    || (typeof dataOptions.submitForm == "undefined" && options.submitForm)
+                    || (typeof dataOptions.submitForm == "undefined" && typeof options.submitForm == "undefined" && $.confirm.options.submitForm)
+                ) {
```

Comment:
Could this be simplified to `dataOptions.submitForm || options.submitForm || $.confirm.options.submitForm` ?